syntax/ps1.vim

```diff
@@ -93,7 +93,7 @@ syn region ps1String start=/@"$/ end=/^"@/ contains=@ps1StringSpecial,@Spell
 syn region ps1String start=/@'$/ end=/^'@/
 
 " Interpolation
-syn match ps1Escape /`./ contained
+syn match ps1Escape /`./    " NOT contained - you can escape quotes to prevent quoting.
```

Comment:
Get rid of the comment (really only makes sense if you know it used to be `contained`) and I think this makes sense.